### PR TITLE
ci: add github action for tests

### PR DESCRIPTION
**Pull request by ahayzen-kdab**

### CODE REVIEW

**Comment by ahayzen-kdab** [comment]
You can see the CI running and passing on my fork here https://github.com/ahayzen-kdab/clang-format-rs/actions

Not sure why it doesn't appear here, hopefully once it's merged it'll start appearing here too.

File: README.md [hunk 1]
```diff
@@ -28,16 +28,6 @@ fn main() {
 }
 ```
 
-# Tests
```
Comment (LeonMatthesKDAB) [inline README.md:31]: This info is now moved into the tests.sh script, so maybe refer to that here in the README.

**Review by LeonMatthesKDAB — COMMENTED** [review]
Good change overall, but I think we should be more explicit about having moved the tests into the `tests.sh` file

File: README.md (hunk 1 above)
Comment (ahayzen-kdab) [reply README.md:31]: Added note about the `tests.sh` script  :-)

**Review by LeonMatthesKDAB — APPROVED** [review]
LGTM

**Pull request merged**